packages/cli/src/routeGeneration/templates/hapi/hapi.hbs

```diff
@@ -1,7 +1,8 @@
 /* tslint:disable */
 /* eslint-disable */
 // WARNING: This file was auto-generated with tsoa. Please do not modify it. Re-run tsoa to re-generate this file: https://github.com/lukeautry/tsoa
-  import { Controller, ValidationService, FieldErrors, ValidateError, TsoaRoute, HttpStatusCodeLiteral, TsoaResponse, fetchMiddlewares } from '@tsoa/runtime';
+import { TsoaRoute, fetchMiddlewares } from '@tsoa/runtime';
+import { HapiTemplateService } from '@tsoa/cli';
```

Comment:
Same here. These services must be (typed) in the runtime package

Reply:
Resolved @ #1574